### DISCUSSION TITLE
posts/2023/07/can-not-say/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#15] [General]
# posts/2023/07/can-not-say/

我们已见面这么多回。

https://guozheng.rbind.io/posts/2023/07/can-not-say/

**Comment by CyrusYip** [comment]
暗恋令人悲喜交加，见面就开心，还没追到就伤心。

**Reply by residualsun1** [reply]
不追了(இдஇ; )打不死我的终究会使我更加强大，我要独自灿烂。

**Comment by liechi** [comment]
换位思考，如果一个你觉得是普通朋友的女孩子对你流露爱意，你会怎么做？不巧的事常有，她不一定不好，只是你没有感觉。

如果没有正式表白的话，找个机会坦率聊一次也好。我个人觉得宁愿失败，不要遗憾。不管结果如何，勇敢面对，然后大步走向属于自己的幸福。

得到明确的结果有个好处，就是可以坚定自己的心意。不然现在决心虽然下了，回头别人无意间一个低眉浅笑就又动摇了。

**Reply by residualsun1** [reply]
> 换位思考，如果一个你觉得是普通朋友的女孩子对你流露爱意，你会怎么做？不巧的事常有，她不一定不好，只是你没有感觉。

嗯，我想答案或许会和现在是一样的。

> 得到明确的结果有个好处，就是可以坚定自己的心意。不然现在决心虽然下了，回头别人无意间一个低眉浅笑就又动摇了。

想起来也是羞愧，之前还大言不惭地说想开了( _ _ )……

**Reply by liechi** [reply]
> 想起来也是羞愧，之前还大言不惭地说想开了( _ _ )……

不必羞愧，这是正常的。无欲才刚。青年情窦初开，遇见单纯的美好，心心念念，看似触手可及却又隔之天远，内心伤感脆弱几乎是必然的。好在心碎过几次就坚强了。勇敢点。

按我初浅的经验，如果你能坚强些，洒脱些，不要过于拘谨，胜算会比现在这样大。

**Reply by residualsun1** [reply]
> 好在心碎过几次就坚强了。勇敢点。

过去想法反反复复。这回要将近一个多月不见，念头可能又是暂时通达了一点。我也不知道是否有几率变得更坚强一点，但执着多几次没结果一定会松开的。感谢你的宽慰，让我不自觉想到黑塞《彷徨少年时》里的德米安和辛克莱。

> 按我初浅的经验，如果你能坚强些，洒脱些，不要过于拘谨，胜算会比现在这样大。

经历使得我一度接触到的概念和话语变得越发鲜活，我相信会逐渐知行合一的。

**Reply by liechi** [reply]
爱恋是个先验且自我强化的过程，在离开的这段时间把注意力转向其他有意思的人和事会有助于打破正向强化的循环。不要期待短暂的分别可以让人通达，有时离别只会让重逢更动人，心颤如在春月下踏着落花（老舍语）。

我没有看过黑塞，但大概能猜到你的比喻想说的意思。跟你的讨论让我想起多年前的一些令人怀念的人和事，所以话会不自觉地多起来。时迁事异，我说的并不一定适合你，可以看作平行世界里另一个故事主人公的体会，有部分或可对你有参考价值。
